app/Console/Commands/FilesystemSetupCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use Bouncer;
+use Illuminate\Console\Command;
+use Kanvas\Apps\Repositories\AppsRepository;
+
+class FilesystemSetupCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'filesystem:disk';
+
+    /**
+     * The console command description.
+     *
+     * @var string|null
+     */
+    protected $description = 'Command that setup the filesystem disk';
+
+    public function handle(): void
+    {
+        $app = AppsRepository::findFirstByKey(env('KANVAS_APP_ID'));
```

Comment:
@arfenis remeber we can't use env need to use config

Also call the app container vía app(Apps:class)